app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -119,6 +119,24 @@ def self.default_blacklisted_event_names
     Settings.ems.ems_amazon.blacklisted_event_names
   end
 
+  def self.provider_create_params
+    {
+      self.name => params_for_create
+    }
+  end
```

Comment:
Ok, now I get it, I don't think we actually need this. I can simply work with the one below. Or is there any special reason for this format?

Reply:
Nope I was just trying to maintain the same format that would be returned by calling it on the base class.

Reply:
Nah, it might make sense to have it on the `ExtManagementSystem` but not here.

Reply:
:+1: